FlowCrypt/Core/Core.swift

```diff
@@ -147,6 +147,12 @@ final class Core: KeyDecrypter {
         }
     }
 
+    func gmailBackupSearch(for email: String) -> String? {
+        let response = try? call("gmailBackupSearch", jsonDict: ["acctEmail": email], data: nil)
+        let result = try? response?.json.decodeJson(as: GmailBackupSearchResponse.self)
+        return result?.query
+    }
```

Comment:
For consistency,  I'd probably write this same as the other methods without the question marks, and return a `String`.

If this is inapropriate, then we should (in another PR) rethink error handling in the whole class.